Tweak backend (#110)
openscm/scmdata: docs/source/data.rst

The **ScmRun** class
--------------------

The :class:`scmdata.ScmRun` class represents a collection of timeseries data including metadata and provides methods for manipulating the data.
Internally, :class:`ScmRun <scmdata.run.ScmRun>` stores the timeseries data in a single :class:`pandas.DataFrame` and the timeseries metadata

> in :class:`pandas.Categorial`, for efficient indexing.

Reviewer: ```suggestion
in :class:`pandas.MultiIndex` of type `pandas.Categorical`, for efficient indexing.
```

I think... I'm not fully across exactly what classes we're using.

This class is the primary way of handling timeseries data within the **scmdata** package.

> For example, the :class:`scmdata.timeseries.TimeSeries`'s can be filtered to only find the :class:`TimeSeries <scmdata.timeseries.TimeSeries>`  which have a ``"scenario"`` metadata label equal to ``"green"`` (see :meth:`scmdata.ScmRun.filter <scmdata.run.ScmRun.filter>` for full details).

Reviewer: ```suggestion
For example, the :class:`scmdata.run.ScmRun` can be filtered to only find the subset of data which have a ``"scenario"`` metadata label equal to ``"green"`` (see :meth:`scmdata.ScmRun.filter <scmdata.run.ScmRun.filter>` for full details).
```

Other operations include grouping, setting and (basic) plotting.

The complete set of manipulation features can be found in the documentation pages of :class:`ScmRun <scmdata.run.ScmRun>`.

:class:`ScmRun <scmdata.run.ScmRun>` has three key properties and one key method, which allow the user to quickly access their data in more standard formats:

* :attr:`values <scmdata.run.ScmRun.values>` returns all of the timeseries as a single :obj:`numpy.ndarray` without any metadata or indication of the time axis.
* :attr:`meta <scmdata.run.ScmRun.meta>` returns all of the timeseries' metadata as a single :obj:`pandas.DataFrame`. This allows users to quickly have an overview of the timeseries held by :class:`scmdata.ScmRun` without having to also view the data itself.

> * :attr:`metadata <scmdata.run.ScmRun.metadata` stores run-specific metadata, metadata which isn't tied to any timeseries specifically.

Reviewer: ```suggestion
* :attr:`metadata <scmdata.run.ScmRun.metadata` stores run-specific metadata i.e. metadata which isn't tied to any timeseries specifically.
```
